tests/test_009_layers.py

```diff
@@ -159,6 +158,59 @@ def test_set_minimum_layer_thickness(plot=False):
         plot_test(regis, ds_new)
 
 
+def test_calculate_transmissivity():
+    regis = get_regis_horstermeer()
+    nlmod.layers.calculate_transmissivity(regis)
+
+
+def test_calculate_resistance():
+    regis = get_regis_horstermeer()
+    # with the default value of between_layers=True
+    nlmod.layers.calculate_resistance(regis)
+    # and also with between_layers=False
+    nlmod.layers.calculate_resistance(regis, between_layers=False)
+
+
+def test_get_layer_of_z():
+    regis = get_regis_horstermeer()
+    z = -100
+
+    layer = nlmod.layers.get_layer_of_z(regis, z)
+
+    assert (regis["botm"].isel(layer=layer) < z).all()
+    top = regis["botm"] + nlmod.layers.calculate_thickness(regis)
+    assert (top.isel(layer=layer) > z).all()
+
+
+def test_aggregate_by_weighted_mean_to_ds():
+    regis = get_regis_horstermeer()
+    regis2 = regis.copy(deep=True)
+
+    # botm needs to have the name "bottom'
+    regis2["bottom"] = regis2["botm"]
```

Comment:
Yeah, this should be fixed in the aggregate method. Let's make it a new issue.

Reply:
ok, I see you did this. Thanks!